### ISSUE TITLE
tape training on fluorescence-dataset seemingly yield bad predictions

### ISSUE THREAD

**Issue by ekvall93** [songlab-cal/tape#103]
Hi,

I have downloaded the fluorescence dataset and wanted to try out the tape-model. I have trained the model with the following command:

`tape-train transformer fluorescence --batch_size 512 --learning_rate 0.0001 --fp16 --warmup_steps 1 --gradient_accumulation_steps 32
`
The best model seems to land around validation error around ~0.7.

When running the eval with: 

`tape-eval transformer fluorescence --batch_size 256 ./results/fluorescence_transformer_21-04-04-10-45-01_547467
`
I get interesting results. All the predictions are always around ~3.19 which most of the time quite far away from the true-values.

Is this as expected, or have I trained the model in the wrong way? 

Best regards,

Markus Ekvall

**Comment by yuanenming** [comment]
I think your settings are quite buggy. Your real batch_size is 512*32=16384, which is infeasible.

BTW, it seems hard to reproduce the results of downstream tasks using this PyTorch version.

**Comment by yuanenming** [comment]
Additionally, you did not declare to use a pre-trained model, I think.

**Comment by rmrao** [comment]
Yup, looks like @yuanenming has identified the issues! Please open a new issue if that doesn't fix it.

**Comment by ekvall93** [comment]
Great, it seems to work now. Thanks

**Comment by bright2311** [comment]
> Great, it seems to work now. Thanks

Hello. I meet the similar confusion when I train the  fluorescence(GFP) task using transformer.The mse loss is even worse than the simple CNN. 
How do you fix it? Thanks.
